### DISCUSSION TITLE
How Exclude A camera in HA DWB setup?

### DISCUSSION THREAD

**Discussion by sjmotew** [mrlt8/docker-wyze-bridge#1071] [General]
How do I exclude a camera from DWB in HA?  I have a battery Wyze cam and since it doesn't use the same SDK, it just shows as blank with constant error on log.

**Comment by mrlt8** [comment]
You should be able to filter-in/out cameras by camera model under by toggling `Show unused optional configuration options` in the add-on config. You can use the `Invert Filter`  option if you want to block the camera models instead of allowing them.
